src/System.Device.Gpio/Interop/Unix/libgpiod/Interop.libgpiod.cs

```diff
@@ -110,6 +110,13 @@ internal partial class libgpiod
         [DllImport(LibgpiodLibrary)]
         internal static extern void gpiod_line_release(IntPtr lineHandle);
 
+        /// <summary>
+        /// Release a previously reserved line.
+        /// </summary>
+        /// <param name="lineHandle">GPIO line handle</param>
+        [DllImport(LibgpiodLibrary)]
+        internal static extern void gpiod_line_release(SafeLineHandle lineHandle);
```

Comment:
Looks like above we already have this method which takes in an IntPtr. Can we modify that one to instead take a SafeLineHandle and fix the callsites to use that instead of IntPtr?

Reply:
The other overload is used internally by the SafeLineHandle itself, therefore I think both are needed.

Reply:
I think this method should only ever be used by the Dispose method in SafeHandle, and never called upopn directly. Take a look at my comment on @krwq's comment where you are using this. We should either not be calling release from there as it could be a bug, or if we decide that that is correct, then we should be calling it via disposing the handle.